src/plinder/data/pipeline/utils.py

```diff
@@ -537,3 +540,111 @@ def create_nonredundant_dataset(*, data_dir: Path) -> None:
     df_trainable_nonredundant.to_parquet(
         data_dir / "index" / "annotation_table_nonredundant.parquet", index=False
     )
+
+
+def pack_linked_structures(data_dir: Path, code: str) -> None:
+    """
+    Pack generated linked structures into a zip file for a particular
+    two character code.
+
+    Parameters
+    ----------
+    data_dir : Path
+        plinder root dir
+    code : str
+        two character code
+    """
+    (data_dir / "links").mkdir(exist_ok=True, parents=True)
+    with ZipFile(
+        data_dir / "links" / f"{code}.zip", "w", compression=ZIP_DEFLATED
+    ) as archive:
+        for search_db in ["apo", "pred"]:
```

Comment:
should also be parameterized with `search_dbs`?

Reply:
Due to how the linked structure generation runs, it is slightly more convenient when making the `{two_char_code}.zip` archives, to lay them out like
```bash
{xx}.zip
    apo/
        system_id/
    pred/
        system_id/
```
rather than 
```bash
apo/
    {xx}.zip/
        system_id/
pred/
    {xx}.zip/
        system_id/
```
so "hard-coding" it here ensures the archives always get made with the union of generated linked structures. Then the `PlinderSystem` abstracts the heterogeneity of this layout vs. e.g. the "standard" entries / systems archives.

Reply:
agreed! just meant like if cross-docking structures are run I guess it would be ["apo", "pred", "holo"]?

Reply:
Yes that's a good point. I had previously been hijacking the `sub_databases` config parameter for every step that needs parameterization over search database. This should be split up for the relevant stage in the pipeline!